app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -195,16 +203,33 @@ fun ConversationScreen(
         conversationCallViewState = conversationCallViewModel.conversationCallViewState,
         conversationInfoViewState = conversationInfoViewModel.conversationInfoViewState,
         conversationMessagesViewState = conversationMessagesViewModel.conversationViewState,
-        onOpenProfile = conversationInfoViewModel::navigateToProfile,
-        onMessageDetailsClick = conversationMessagesViewModel::openMessageDetails,
+        onOpenProfile = {
+            with(conversationInfoViewModel) {
+                val (mentionUserId: UserId, isSelfUser: Boolean) = mentionedUserData(it)
+                if (isSelfUser) navigator.navigate(NavigationCommand(SelfUserProfileScreenDestination))
+                else (conversationInfoViewState.conversationDetailsData as? ConversationDetailsData.Group)?.conversationId.let {
+                    navigator.navigate(NavigationCommand(OtherUserProfileScreenDestination(mentionUserId, it)))
+                }
+            }
+        },
+        onMessageDetailsClick = { messageId: String, isSelfMessage: Boolean ->
+            appLogger.i("[ConversationScreen][openMessageDetails] - isSelfMessage: $isSelfMessage")
```

Comment:
Do we need this logger?

Reply:
This one was in the ViewModel previously, bound to this action, not sure if we use that log but I moved it as well to not lose it in logs